File: PetHealth.DAL/Entities/Clinic.cs
using System.Collections.Generic;

namespace PetHealth.DAL.Entities
{
    public class Clinic
    {
        public int Id { get; set; }

        public User User { get; set; }

        public int UserId { get; set; }

        public string Name { get; set; }

        public IEnumerable<ClinicPet> Pets { get; set; }

        public IEnumerable<Comment> Comments { get; set; }

        public IEnumerable<AvailableTime> AvailableTimes { get; set; }

        public IEnumerable<Chat> Chats { get; set; }

        public IEnumerable<Visit> Visits { get; set; }
    }
}

File: pethealth_ui/src/app/types/Clinic.ts
import { ClinicComment } from "./Comment";

export interface Clinic extends ClinicShort {
  imageUrl?: string;
  comments: Array<ClinicComment>;
  amountOfPets: number;
  user: string;
}

export interface ClinicShort {
  id: number;
  name: string;
}
